File: packages/kevin_flutter_accounts/ios/Classes/Entity/AccountSessionConfigurationEntity.swift
internal struct AccountSessionConfigurationEntity: Codable {
    let state: String
    let preselectedCountry: String?
    let disableCountrySelection: Bool
    let countryFilter: [String]
    let bankFilter: [String]
    let preselectedBank: String?
    let skipBankSelection: Bool
    let accountLinkingType: String
    
    init(
        state: String,
        preselectedCountry: String?,
        disabledCountrySelection: Bool,
        countryFilter: [String],
        bankFilter: [String],
        preselectedBank: String?,
        skipBankSelection: Bool,
        accountLinkingType: String)
    {
        self.state = state
        self.preselectedCountry = preselectedCountry
        self.disableCountrySelection = disabledCountrySelection
        self.countryFilter = countryFilter
        self.bankFilter = bankFilter
        self.preselectedBank = preselectedBank
        self.skipBankSelection = skipBankSelection
        self.accountLinkingType = accountLinkingType
    }
}

File: packages/kevin_flutter_accounts/kevin_flutter_accounts_android/android/src/main/kotlin/eu/kevin/flutter/accounts/entity/AccountSessionConfigurationEntity.kt
package eu.kevin.flutter.accounts.entity

import kotlinx.serialization.Serializable

@Serializable
internal data class AccountSessionConfigurationEntity(
    val state: String,
    val preselectedCountry: String?,
    val disableCountrySelection: Boolean,
    val countryFilter: List<String>,
    val bankFilter: List<String>,
    val preselectedBank: String?,
    val skipBankSelection: Boolean,
)